canonical-web-and-design/maas-ui
# fix(single-spa): fix for the ui not loading in Safari

## Done

- Fix a regex that was causing the UI to not display in Safari.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Run the app with `yarn start` so that you are viewing the site with single-spa.
- Open the ui in Safari, it should load.
- Double check that it still works in Firefox/Chrome.

## Fixes

Fixes: #3045.
